### PR TITLE
[Blog] Top  5 things for Azure Developers: Introduction 

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

File: themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+## Let's Get Started!
+
+So here are the top 5 things essential to know as an Azure developer:
+
+- configuring and deploying virtual machines
+- developing and deploying Azure Functions
+- deploying static websites on Azure
+- configuring Azure Kubernetes Service and deploying containerized applications
+- using Azure DevOps to automating building and deployment of infrastructure and applications
```
Comment (mattstratton) [inline themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md:112]:
```suggestion
- using Azure DevOps to automate the building and deployment of infrastructure and applications
```

File: themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+Azure provides two ways to deploy static websites. Azure let's you deploy a static web application with Azure App Service and the recently released Azure Static Web Apps. Azure App Service deploys websites by uploading the website to Azure storage configured to serve web content. Azure App Service automatically builds and deploys full stack web apps to Azure from a code repository, which is follows the [Jamstack](https://jamstack.org/) model for developing and deploying websites.
+
+We'll look at both methods and discuss the pros and cons of both methods.
+
+## Kubernetes
+
+Kubernetes is the leading container orchestration service. Azure Kubernetes Service (AKS) is a hosted service that can deploy, scale, and manage containerized applications in a cluster. AKS fulfills many different use cases, for example:
+
+- If you have an existing application, think n-tier applications, you can containerize the components and "lift and shift" your application to run in Kubernetes.
+- You can deploy microservices on AKS to horizontal scaling, load balancing, secrets management, and self-healing.
```
Comment (mattstratton) [inline themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md:63]:
```suggestion
- You can deploy microservices on AKS to add horizontal scaling, load balancing, secrets management, and self-healing.
```

File: themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+![Serverless function](./image2.png)
+
+## Static websites
+
+Static websites are quickly becoming the defacto way to publish content. Built from libraries and frameworks that don't use server side rendering like React, Angular, or Vue, they include HTML, CSS, image assets, and Javascript in the application. Instead of serving assets from a server or servers scaled horizontally, the static assets are distributed through a Content Delivery Network (CDN). This reduces latency, resulting in faster websites and increased customer satisfaction.
+
+Azure provides two ways to deploy static websites. Azure let's you deploy a static web application with Azure App Service and the recently released Azure Static Web Apps. Azure App Service deploys websites by uploading the website to Azure storage configured to serve web content. Azure App Service automatically builds and deploys full stack web apps to Azure from a code repository, which is follows the [Jamstack](https://jamstack.org/) model for developing and deploying websites.
```
Comment (mattstratton) [inline themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md:54]:
```suggestion
Azure provides two ways to deploy static websites. Azure lets you deploy a static web application with Azure App Service and the recently released Azure Static Web Apps. Azure App Service deploys websites by uploading the website to Azure storage configured to serve web content. Azure App Service automatically builds and deploys full stack web apps to Azure from a code repository, which follows the [Jamstack](https://jamstack.org/) model for developing and deploying websites.
```

File: themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+Azure provides Infrastructure as a Service (IaaS) for deploying your applications on either Linux or Windows VMS. The IaaS include the necessary support such as Azure Virtual Network and storage. As with any IaaS solution, you are responsible for installing software, configuration, and maintenance of your application and any maintenance the VM might require, such as operating system patches.
+
+If your application doesn't fit into a Platform as a Service (PaaS) model, consider using virtual machines to provide the fine-grained control required for your deployment.
+
+![VM in Azure Portal](./image3.png)
+
+## Serverless
+
+At the opposite end of cloud infrastructure is serverless computing. Serverless eliminates the need for managing infrastructure and lets you write and deploy code. Azure automatically provisions, scales, and manages the infrastructure. You can write code that runs in response to events or on a schedule.
```
Comment (mattstratton) [inline themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md:42]:
```suggestion
At the opposite end of cloud infrastructure is serverless computing. Serverless eliminates the need to manage infrastructure and lets you focus on writing and deploying code. Azure automatically provisions, scales, and manages the infrastructure. You can write code that runs in response to events or on a schedule.
```

File: themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+With over 200 services, knowing which one to use can be confusing. Deciding where to host your application is the first step. This series of articles covers the most commonly used Azure services such as virtual machines, serverless functions, static websites, Kubernetes, and Azure App Service.  In addition, we'll cover how to deploy applications using Azure DevOps.
+
+In this article, we'll match use cases to cloud services to help you navigate Azure.
+
+## Azure Virtual Machines
+
+A common cloud migration path is to use virtual machines if you have existing infrastructure or on-premise applications. It is especially true if you want complete control over the infrastructure and application environment.
+
+Azure provides Infrastructure as a Service (IaaS) for deploying your applications on either Linux or Windows VMS. The IaaS include the necessary support such as Azure Virtual Network and storage. As with any IaaS solution, you are responsible for installing software, configuration, and maintenance of your application and any maintenance the VM might require, such as operating system patches.
```
Comment (mattstratton) [inline themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md:34]:
```suggestion
Azure provides Infrastructure as a Service (IaaS) for deploying your applications on either Linux or Windows VMs. IaaS includes all necessary support, such as Azure Virtual Network and storage. As with any IaaS solution, you are responsible for the installation of software, configuration, and maintenance of your application and any maintenance the VM might require, such as operating system patches.
```

File: themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+We'll examine these options and do a deep dive on application deployment in a follow-up article.
+
+## DevOps integration
+
+Before Continuous integration (CI) and Continuous Deployment (CD) became prevalent, applications were manually built, compiled, and maintained in a large codebase. CI/CD automation lets you build quickly, test, and deploy services. Both application fixes and new features are shipped quickly to your customers, and you can rapidly respond to changing customer demands. For example, if demand increases, you can quickly add resources to your application or add a serverless function that increases and decreases resources as needed.
+
+![Devops architecture](./image1.png)
+
+If you're deploying applications on Azure Functions, virtual machines, AKS, or provisioning infrastructure, the build process should be. Azure offers a range of services such as Azure Container Registry, AKS, and Azure Cosmos DB that can integrate with popular CI/CD pipelines like Azure DevOps, Jenkins, Github, and TeamCity.
```
Comment (mattstratton) [inline themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md:100]: > the build process should be.

should be...what? :)

**Review by mattstratton — CHANGES_REQUESTED** [review]
A few small changes suggested, and one sentence ends incomplete

File: themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md (hunk 6 above)
Comment (spara) [reply themes/default/content/blog/top-5-things-for-azure-devs-intro/index.md:100]: automated

**Review by mattstratton — APPROVED** [review]

**Pull request merged**